packages/flutter/lib/src/gestures/events.dart

```diff
@@ -782,6 +773,7 @@ class PointerUpEvent extends PointerEvent {
     Offset position = Offset.zero,
     int buttons = 0,
     bool obscured = false,
+    // PointerUpEvent can contain non-zero pressure. https://github.com/flutter/flutter/pull/30874
```

Comment:
let's file an issue to track this (and you can link to that here rather than the PR)

Reply:
(in due course we should probably change this, but i'm fine with doing that in a later PR)

Reply:
Changed to https://github.com/flutter/flutter/issues/31340